core/chains/evm/txmgr/eth_resender.go

```diff
@@ -112,11 +117,25 @@ func (er *EthResender) resendUnconfirmed() error {
 		if err != nil {
 			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
 		}
+		oldestAttempt, exists := findOldestUnconfirmedAttempt(attempts)
```

Comment:
Move this complete logic to inside a different function, so it is easier for readability, that this is all just logging, and no actual logic.